app/src/main/java/com/github/multimatum_team/multimatum/viewmodel/GroupViewModel.kt

```diff
@@ -104,11 +105,14 @@ class GroupViewModel @Inject constructor(
      * @pram id the ID of the group to rename
      * @param newName the new name of the group
      */
-    fun renameGroup(id: GroupID, newName: String) =
-        viewModelScope.launch {
-            groupRepository.rename(id, newName)
-            LogUtil.debugLog("renaming group with id $id to $newName")
-        }
+    fun renameGroup(id: GroupID, newName: String) = viewModelScope.launch {
+        groupRepository.rename(id, newName)
+        LogUtil.debugLog("renaming group with id $id to $newName")
```

Comment:
It'ld be to remove log when you're not using them anymore, since this can lead to messy log console.